tdms/src/iterator.cpp

```diff
@@ -871,10 +871,10 @@ void execute_simulation(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs
 
   //work out if we have any disperive materials
   bool is_disp = is_dispersive(materials, gamma, params.dt, I_tot, J_tot, K_tot);
-  //work out if we have conductive background
-  bool is_cond = is_conductive(rho_cond, I_tot, J_tot, K_tot);
-  //work out if we have a dispersive background
-  if (params.is_disp_ml) params.is_disp_ml = is_dispersive_ml(ml, K_tot);
+  //work out if we have conductive background: background is conductive if at least one entry exceeds 1e-15
+  bool is_cond = !(rho_cond.all_elements_less_than(1e-15, I_tot + 1, J_tot + 1, K_tot + 1));

```

Comment:
Would you mind renaming that symbol to `bool is_conductive`?

Reply:
And (for future refactorings) does it make sense to collect all of the background parameters(?) together?
Then we have a more self-evident line of code like...
```c++
bool is_conductive = Background.is_conductive(...);
```
...or whatever.